crates/accelerate/src/synthesis/clifford/utils.rs

```diff
@@ -227,6 +227,24 @@ impl Clifford {
                     clifford.append_s(qubits[0].index());
                     Ok(())
                 }
+                StandardGate::SdgGate => {
+                    clifford.append_sdg(qubits[0].0 as usize);
+                    Ok(())
+                }
+                StandardGate::SXGate => {
+                    // todo: replace by a single update
```

Comment:
this can be done already in this PR, see:
https://github.com/Qiskit/qiskit/blob/db0433920c86e55f4ec06b3837d10d94b040e117/qiskit/quantum_info/operators/symplectic/clifford_circuits.py#L325

Reply:
Thanks, I planned to do it, but forgot. Added in [41ae2f3](https://github.com/Qiskit/qiskit/pull/13301/commits/41ae2f33146762e3ec0b0e456d3f63dc40e0d0c9).